src/s1reader/s1_etad.py

```diff
@@ -0,0 +1,202 @@
+"""Wrapper of s1etad module to read S1-ETAD products."""
+import datetime
+import glob
+import os
+
+import numpy as np
+from matplotlib import pyplot as plt
+from scipy.interpolate import RectBivariateSpline
+
+try:
+    import s1etad
```

Comment:
I am a bit confused here. Are we planning to add this dependency? If yes, please update the list of dependency for the s1-reader. In addition, I thought that our goal was to compute the corrections from scratch. Are we adding support for ETAD corrections files for debug purposes?

Reply:
>  I thought that our goal was to compute the corrections from scratch. Are we adding support for ETAD corrections files for debug purposes?

That's my understanding currently. So I did not update the dependency of `s1reader`.

Reply:
I think it is a bit of a contradiction to support reading ETAD datasets but not including the dependency necessary to read them. I am not sure how to go about this as the only occasion I see this be handy is for debugging purposes (i.e. compare our in-house developed correction layers with that from ESA/ETAD). Is the `s1etad` dependency a light one? If this is the case, probably, we should just add it. What is your take on this?

Reply:
As you have said, this module is for testing purposes only. The `s1etad` module dependency is not light. Thus, I like the current way of not adding it to our requirements.txt file. Users of this script is assume to be advanced, thus, know how to install s1etad, based on the printout msg.